src/pudl/metadata/fields.py

```diff
@@ -1846,7 +1846,11 @@
         "description": "Utility name, chosen arbitrarily from the several possible utility names available in the utility matching process. Included for human readability only.",
     },
     "utility_owned_capacity_mw": {"type": "number", "unit": "MW"},
+    "utility_plant_asset_type": {"type": "string"},  # description
+    "utility_plant_value": {"type": "number", "description": "Utility plant value."},
     "utility_pobox": {"type": "string"},
+    "utility_type": {"type": "string"},  # enum me + description
```

Comment:
Did we decide to hold off on adding the enumerations for these columns until we can get them directly from the metadata / parameterizations?

Reply:
added description but not enums bc yes!